modules/host/storageobligations.go

```diff
@@ -414,6 +404,22 @@ func (h *Host) managedAddStorageObligation(so storageObligation) error {
 		h.log.Println("Error with transaction set, redacting obligation, id", so.id())
 		return composeErrors(err, h.removeStorageObligation(so, obligationRejected))
 	}
+
+	// Increase contract count by one only if this is a new filecontract.
+	if len(so.SectorRoots) == 0 {
+		h.financialMetrics.ContractCount++
+	}
```

Comment:
I'm not sure about this part. I guess it depends on the precise meaning of `ContractCount`. Even if you renew a contract, the host still needs to track the old contract until it ends, so it might make sense to increment `ContractCount` even for renewed contracts.

Reply:
You are right! I reverted the commits. Was confused about the 'renewal' word, where it actually is a new storage obligation we need to track.